test/testinput/bufrtest_python_test.py

```diff
@@ -160,16 +160,35 @@ def test_highlevel_add():
     assert np.allclose(obs_temp, obs_orig)
     assert obs_temp.shape == data.shape
 
+def test_highlevel_append():
+    DATA_PATH = 'testinput/data/gdas.t00z.1bhrs4.tm00.bufr_d'
+    YAML_PATH = 'testinput/bufrtest_hrs_basic_mapping.yaml'
+
+    container = bufr.Parser(DATA_PATH, YAML_PATH).parse()
+
+
+    new_container = bufr.DataContainer()
+    new_container.append(container)
+    new_container.append(container)
```

Comment:
@rmclaren I did no check your PR in time.  I have one question.
Why do we need tow append the container twice to the new_container?

Reply:
@emilyhcliu I guess this is testing appending one container into another one, though in the testcase these two containers are the same.

Reply:
It's just a test. So I start with an empty container, and then append the same data to 2 times. Later I check to make sure that is what actually happened (make sure appending the same data twice worked as expected).